collie/model/base/trainer.py

```diff
@@ -176,12 +176,21 @@ class CollieMinimalTrainer():
     max_depth: int
         Maximum depth of modules to show, use -1 to show all modules or 0 to show no summary.
         Related to PyTorch Lightning's ``Trainer.enable_model_summary`` argument.
+    enable_model_summary: bool
+        Whether to enable or disable the model summarization. Defaults to True.
```

Comment:
No need to put the defaults in the docstring! 

```suggestion
        Whether to enable or disable the model summarization
```